command/cp.go

```diff
@@ -528,7 +528,9 @@ func (c Copy) doDownload(ctx context.Context, srcurl *url.URL, dsturl *url.URL)
 		return err
 	}
 
-	file, err := dstClient.Create(dsturl.Absolute())
+	dstPath := filepath.Dir(dsturl.Absolute())
+	dstFile := filepath.Base(dsturl.Absolute())
+	file, err := dstClient.CreateTemp(dstPath, dstFile)
```

Comment:
```suggestion
	file, err := dstClient.CreateTemp("", dstFile)
```

lets do not give any path to keep cwd clean as @seruman suggested

Reply:
Creating the temporary file in cwd can improve performance when the temp folder and target path are on different drives or partitions. Otherwise, it will need to make a copy of the temp file, which can take a long time for large files.